### PR TITLE
Add a CI with GitHub Actions

### LINKED ISSUE #59: Run test suite through github action

### PR DESCRIPTION
**Pull request by HedicGuibert**
Fix #59

### CODE REVIEW

**Comment by lyrixx** [comment]
Don't forget to remove the travis file

**Review by damienalexandre — CHANGES_REQUESTED** [review]
Nice :) 

Could you add more PHP version? This lib use the XML parser and it's very sensible ^^

https://github.com/jolicode/JoliTypo/blob/master/.travis.yml#L3-L7

File: .github/workflows/setup.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.1', '7.2', '7.3', '7.4', '8.0']
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup PHP 7.4
```
Comment (lyrixx) [inline .github/workflows/setup.yml:20]: The name is not correct when installing php 7.1 for example

File: .github/workflows/setup.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
+name: install project and run tests
```
Comment (lyrixx) [inline .github/workflows/setup.yml:2]:
```suggestion
name: Install project and run tests
```

**Review by lyrixx — APPROVED** [review]
Thanks 👍🏼

**Pull request merged**